can_tools/scrapers/official/federal/CDC/cdc_testing_cases.py

```diff
@@ -74,3 +74,11 @@ def normalize(self, data: pd.DataFrame) -> pd.DataFrame:
                 locations_to_drop=[2066, 2063, 72888, 72999],
             ).pipe(self._reshape_variables, variable_map=self.variables)
         )
+
+        # There are duplicated rows and multiple rows for a single variable in the dataset.
+        data = data.drop_duplicates()
+        # If a variable (e.g. a location, date, and variable type) has multiple entries just select the most
+        # recent occurrence. There does not seem to be a pattern or a discernably "correct" value.
```

Comment:
How confident are we that this will work out to give us valid data?  Should we proactively build a snapshot with this and validate it today before tomorrow's deploy?

Reply:
I'm fairly confident it will be valid--the values I saw seemed to be pretty close--but good idea, definitely worth building a snapshot before tomorrow